samples/ask-akka-agent/src/main/java/akka/ask/agent/api/AskHttpEndpoint.java

```diff
@@ -48,15 +48,17 @@ public HttpResponse ask(QueryRequest request) {
   @Post("/new-ask") // FIXME
   public String newAsk(QueryRequest request) {
     return componentClient
-        .forAgent(Optional.of(request.sessionId()))
+        .forAgent()
+        .inSession(request.sessionId())
         .method(NewAskAkkaAgent::ask)
         .invoke(request.question);
```

Comment:
This is how the new agent component client looks like. Some peopel found it confusing to have the session id at the same position as entity and workflow id.